src/components/Editable.native.tsx

```diff
@@ -133,20 +130,12 @@ const showDuplicationAlert = duplicateAlertToggler()
 // intended to be global, not local state
 let blurring = false
 
-// eslint-disable-next-line jsdoc/require-jsdoc
-const mapStateToProps = (state: State, props: EditableProps) => {
-  return {
-    isCursorCleared: props.isEditing && state.cursorCleared,
-  }
-}
-
```

Comment:
I tried this before and it seemed like it made the Editable get re-rendered too often. We can try it again but let's make sure it does not introduce additional re-renders. I believe `connect` shields the child component so that it is not re-rendered when the parent is re-rendered.

Since this is not platform specific, it should be applied to both `Editable.native.tsx` and `Editable.tsx`.